acoustid/future/api/app.py

```
@@ -0,0 +1,89 @@
+
+ALLOWED_FINGERPRINT_VERSIONS = frozenset({1})
+
+
+async def handle_submission(request: Request) -> MsgspecResponse:
+    body = await request.body()
+    req = msgspec.json.decode(body, type=SubmissionRequest)
+
+    for submission in req.submissions:
+        try:
+            fp = await asyncio.to_thread(decode_legacy_fingerprint, submission.fingerprint)
+        except Exception:
```

Ruff (0.8.2)

Do not catch blind exception: `Exception`

(BLE001)